packages/create-cloudflare/e2e-tests/frameworks.test.ts

```diff
@@ -468,6 +478,9 @@ const verifyBuildScript = async (
 		[pm, "run", script],
 		{
 			cwd: projectPath,
+			env: {
+				NODE_ENV: "production",
```

Comment:
ooc: why is this needed? 😮

Reply:
This is needed because without it the `NODE_ENV` is set to `test`, which it inherits from the `vitest` call. When `svelte` is built with `NODE_ENV=test` it fails.

Reply:
ah ok, thanks for the clarification 🙂